### PR TITLE
doc: remove outdated messages for custom_protocol about Linux

### LINKED ISSUE #945: Update custom protocol document about Linux
I found we still have the [note](https://docs.rs/wry/latest/wry/webview/struct.WebViewAttributes.html#structfield.custom_protocols) about headers isn't working on Linux.
But this should be fixed by 2.34 already.
We can remove these notes around the doc.

### PR DESCRIPTION
**Pull request by realwakka**
### What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Docs
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

### Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### Checklist
- [x] This PR will resolve #945 
- [ ] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/wry/blob/dev/.changes/readme.md).
- [ ] I have added a convincing reason for adding this feature, if necessary
- [ ] It can be built on all targets and pass CI/CD.

### CODE REVIEW

File: src/webview/mod.rs [hunk 1]
```diff
@@ -126,8 +126,7 @@ pub struct WebViewAttributes {
   /// different Origin headers across platforms:
   ///
   /// - macOS: `<scheme_name>://<path>` (so it will be `wry://examples` in `custom_protocol` example)
-  /// - Linux: Though it's same as macOS, there's a [bug] that Origin header in the request will be
-  /// empty. So the only way to pass the server is setting `Access-Control-Allow-Origin: *`.
+  /// - Linux: Same as macOS.
```
Comment (amrbashir) [inline src/webview/mod.rs]: just change it to `- macOS and Linux: `.

you should also add a note that for headers to work, you need to enable `linux-headers` flag

**Review by amrbashir — CHANGES_REQUESTED** [review]

File: src/webview/mod.rs (hunk 1 above)
Comment (realwakka) [reply src/webview/mod.rs]: Thanks for review! I fixed it

**Comment by amrbashir** [comment]
Oh looks like you didn't sign your commits and I can't merge the PR unless they are signed. You need to setup [commit signing](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits), then you can sign past commit like [this](https://superuser.com/questions/397149/can-you-gpg-sign-old-commits/1123928#1123928) for example.

**Comment by realwakka** [comment]
@amrbashir Oh I didn't know about signature. I made a key and signed to the commits. Thanks for kind links!

**Review by amrbashir — APPROVED** [review]
Thanks for your contribution

**Pull request merged**